src/utils.js

```diff
@@ -242,3 +248,34 @@ export function processData(data, populateCensusTable, totalCatCode) {
   });
   return populateCensusTable;
 }
+
+export function addTotalLevelToMetadata(metadata) {
```

Comment:
the metadata object in this PR has a totalsCode key that could be used here

Reply:
this was created when we thought we would need to do this bit of data handling on the front end, but I'll remove it now that we know we're getting it in more or less this shape from the API